qutebrowser/browser/downloads.py

```diff
@@ -873,6 +946,18 @@ def remove_items(self, downloads):
                 download.deleteLater()
         self.endRemoveRows()
 
+    def update_indexes(self):
+        """Update indexes of all DownloadItems"""
+        first_idx = None
+        for i, d in enumerate(self.downloads, 1):
+            if first_idx is not None and d.index != i:
```

Comment:
If I'm not mistaken, this will never set `first_idx` as it will always be `None`.